enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "2.0.27"
+__version__ = "2.0.28"
```

Comment:
Did we already add release notes for 2.0.28?

Reply:
Yes in a previous PR.

Reply:
https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst#2028---2019-12-3

Reply:
Reference: https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst#2028---2019-12-3